main.js

```diff
@@ -41,21 +41,21 @@ app.get('/enter', function (req, resp) {
             if ((result[0].total_lots - result[0].total_vehicles) === 0) {
 
                 /* There are no free parking lots
-                *  Send request for client's phone number to add to the Waiting list
-                */
+                 *  Send request for client's phone number to add to the Waiting list
+                 */
                 resp.send('client_phone');
             }
             else{
                 // Increase the number of vehicles by 1 and update DB accordingly
                 connection.query('UPDATE garage SET total_vehicles=? WHERE id=1;', [result[0].total_vehicles+1],
```

Comment:
This and the following two `connection.query` calls have callbacks which means they're asynchronous.
This creates a race condition as your third query is dependant on the first two succeeding, this will have unexpected behaviour.  I would suggest wrapping each query with a promise to properly handle the order of these calls as you would expect.